layouts/fastly/helix.vcl

```diff
@@ -2359,11 +2374,14 @@ sub vcl_deliver {
     set req.http.x-last-activation-id = resp.http.x-last-activation-id;
   }
 
-  if (!req.http.X-Debug) {
-    # Unless we are debugging, shut up chatty headers
+  if (!req.http.X-Debug && req.http.X-Request-Type != "Content/LNK") {
```

Comment:
why would be ever unset those? 
right now, the content proxy correctly sets the cors headers. so no need to recompute them here.

Reply:
We are serving from a number of backends that are all opinionated about CORS, including Adobe I/O and GitHub. The generic cleanup is the best approach to deal with that.